cfme/tests/ansible/test_embedded_ansible_automate.py

```diff
@@ -306,3 +309,110 @@ def test_alert_run_ansible_playbook(full_template_vm_modscope, alert_profile, re
             timeout=60)
     except TimedOutError:
         pytest.fail("Ansible playbook method hasn't been executed.")
+
+
+@pytest.fixture(scope='module')
+def setup_ansible_repository(appliance, wait_for_ansible):
+    repositories = appliance.collections.ansible_repositories
+    repository = repositories.create(
+        name="test_playbooks_automate",
+        url=cfme_data.ansible_links.playbook_repositories.embedded_ansible,
+        description=fauxfactory.gen_alpha()
+    )
+    view = navigate_to(repository, "Details")
+    wait_for(
+        lambda: repository.status == "successful",
+        timeout=60,
+        fail_func=view.toolbar.refresh.click
+    )
+    yield
+    repository.delete_if_exists()
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[1678132, 1678135])
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.parametrize(
+    ("import_data", "instance"),
+    ([DatastoreImport("bz_1678135.zip", "Ansible_State_Machine_for_Ansible_stats3",
```

Comment:
Should this be bz_1678132.zip  ?

Reply:
@sshveta I agree, It creates confusion but in this test case. I have automated two different BZs and I got that zip of automate datastores from devel. So I only uploaded that zip file under one BZ(bz_1678135) name. I dont think, it is good idea to upload same data on FTP by two different names.